machine-learning/core-ml-intern.md

```diff
@@ -0,0 +1,47 @@
+---
+authors: @sanchit
+groups: product
+teams: core-ml
+level: NA
+active: true
+---
+# Machine Learning Research Intern
+
+We are looking for an ML Research intern for a duration of 5 months. As an intern, you will be working closely with the core-ml team. You will be involved in owning various product initiatives, creating protypes and delving into research papers. 
+Our team works closely with the research team, broadly on the topics such as Spoken Language Understanding, Speech Recognition, Paralinguistics and Response Production.
```

Comment:
We don't have to say the team, we can make it like the person will work closely with our product and research teams etc.